export.py

```diff
@@ -1779,6 +1779,21 @@ def has_emissive_material(db):
     return False
 
 
+def export_for_bake(db):
+    export = True
+    node_names = []
+    for mat in db.material:
+        if mat and mat.node_tree:
+            nt = mat.node_tree
+            for n in nt.nodes:
+                node_names.append(n.name)
+    if node_names:
+        if 'PxrBakeTexture' not in node_names:
+            export = False
+
```

Comment:
K.  I'll have to rethink how this one ties into the 'get instances and blocks' function.  The reason I set it up the way I did is I was trying to get the data block loop in get instances to only run once and to cover all the bases and make sure important objects are exported (like the camera) I settled with this method.